.github/workflows/pilotWrapper.yml

```diff
@@ -29,3 +29,22 @@ jobs:
         cp tests/Integration/WorkloadManagementSystem/Test_GenerateAndExecutePilotWrapper.py .
         eval "$(conda shell.bash hook)" && conda activate python_${{ matrix.python }}
         python Test_GenerateAndExecutePilotWrapper.py file://${{ github.workspace }}/src/DIRAC/WorkloadManagementSystem/Utilities/PilotWrapper.py
+
+  pylint-py27:
```

Comment:
I understand the logic of having it here in the pilotWrapper workflow, but I think I would prefer to have this block in the basic workflow tests